### ISSUE TITLE
Cannot run "help" command when locale is not UTF-8

### ISSUE THREAD

**Issue by bwenrich** [Backblaze/B2_Command_Line_Tool#888]
I found that, on systems with non-UTF locale like `LANG=en_US` instead of `LANG=en_US.UTF-8`, the `b2 --help` command fails with a traceback about Unicode conversion of the u2022 (bullet) character. 

This has been previously discussed https://github.com/Backblaze/B2_Command_Line_Tool/issues/766 https://github.com/Backblaze/B2_Command_Line_Tool/issues/755 and addressed in https://github.com/Backblaze/B2_Command_Line_Tool/pull/768, but I'm wondering if there is a regression or something else wrong in my environment. I can reproduce the problem on several Ubuntu 22.04 systems, were the only obvious difference is the locale and it's unable to display bullets in the help text such as https://github.com/Backblaze/B2_Command_Line_Tool/blob/f2512b893c43a8c566018551bc26daa7b1518ff5/b2/console_tool.py#L650-L652

```
user@host: ~$ cat /etc/lsb-release | grep -i desc
DISTRIB_DESCRIPTION="Ubuntu 22.04.2 LTS"
user@host: ~$ pip3 install --upgrade b2
user@host: ~$ b2 version
b2 command line tool, version 3.9.0
user@host: ~$ b2
Traceback (most recent call last):
  File "/home/b2backup/.local/bin/b2", line 8, in <module>
    sys.exit(main())
  File "/home/b2backup/.local/lib/python3.10/site-packages/b2/console_tool.py", line 3530, in main
    exit_status = ct.run_command(sys.argv)
  File "/home/b2backup/.local/lib/python3.10/site-packages/b2/console_tool.py", line 3370, in run_command
    args = parser.parse_args(argv[1:])
  File "/usr/lib/python3.10/argparse.py", line 1838, in parse_args
    args, argv = self.parse_known_args(args, namespace)
  File "/usr/lib/python3.10/argparse.py", line 1871, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/usr/lib/python3.10/argparse.py", line 2080, in _parse_known_args
    start_index = consume_optional(start_index)
  File "/usr/lib/python3.10/argparse.py", line 2020, in consume_optional
    take_action(action, args, option_string)
  File "/usr/lib/python3.10/argparse.py", line 1948, in take_action
    action(self, namespace, argument_values, option_string)
  File "/usr/lib/python3.10/argparse.py", line 1111, in __call__
    parser.print_help()
  File "/usr/lib/python3.10/argparse.py", line 2568, in print_help
    self._print_message(self.format_help(), file)
  File "/usr/lib/python3.10/argparse.py", line 2574, in _print_message
    file.write(message)
UnicodeEncodeError: 'latin-1' codec can't encode character '\u2022' in position 1325: ordinal not in range(256)

user@host: ~$ locale
LANG=en_US
LANGUAGE=en_US:
LC_CTYPE="en_US"
LC_NUMERIC="en_US"
LC_TIME="en_US"
LC_COLLATE="en_US"
LC_MONETARY="en_US"
LC_MESSAGES="en_US"
LC_PAPER="en_US"
LC_NAME="en_US"
LC_ADDRESS="en_US"
LC_TELEPHONE="en_US"
LC_MEASUREMENT="en_US"
LC_IDENTIFICATION="en_US"
LC_ALL=

user@host: ~$ LANG=en_US.UTF-8
user@host: ~$ b2 --help
b2 [-h] ## correct output
```

### Expected Behavior
The UTF-8 capability of the environment is detected and uses the fallback if needed. The command `b2` / `b2 --help` displays the program usage information.

### Actual Behavior
The non-UTF-8 environment is not detected. The traceback and error message are displayed

### Workaround
Change the system locale or override the user's settings. This is non-obvious because the error message is not entirely clear on the source of the issue or what settings are acceptable.

### Impact
Unknown how common the non-UTF-8 systems are. I am not intentionally in non-UTF-8 but found it is part of my cloud provider's VM template. The user experience for new users would be improved, as running `b2` or `b2 --help` is likely one of the first things users do after installing the program.

**Comment by ppolewicz** [comment]
It looks like a regression, yes. I'll look into it myself or ask a friend to do it early next week

**Comment by ppolewicz** [comment]
fixed by #891
